code/modules/mob/living/carbon/carbon.dm

```diff
@@ -725,4 +725,80 @@ var/list/ventcrawl_machinery = list(/obj/machinery/atmospherics/unary/vent_pump,
+			if(!selfDrink(toEat))
+				return 0
+		else
+			if(!selfFeed(toEat, fullness))
+				return 0
+	else
+		if(!forceFed(toEat, user, fullness))
+			return 0
+	consume(toEat)
+	return 1
+
+/mob/living/carbon/proc/selfFeed(var/obj/item/weapon/reagent_containers/food/toEat, fullness)
+	if(istype(toEat, /obj/item/weapon/reagent_containers/food/pill))
+		src << "<span class='notify'>You [toEat.apply_method] [toEat].</span>"
+	else
+		if (fullness <= 50)
+			src << "<span class='warning'>You hungrily chew out a piece of [toEat] and gobble it!</span>"
+		else if (fullness > 50 && fullness <= 150)
+			src << "<span class='notice'>You hungrily begin to eat [toEat].</span>"
+		else if (fullness > 150 && fullness <= 350)
+			src << "<span class='notice'>You take a bite of [toEat].</span>"
+		else if (fullness > 350 && fullness <= 550)
+			src << "<span class='notice'>You unwillingly chew a bit of [toEat].</span>"
+		else if (fullness > (550 * (1 + overeatduration / 2000)))	// The more you eat - the more you can eat
+			src << "<span class='warning'>You cannot force any more of [toEat] to go down your throat.</span>"
+			return 0
+	return 1
+
+/mob/living/carbon/proc/selfDrink(var/obj/item/weapon/reagent_containers/food/drinks/toDrink, mob/user)
+	return 1
+
+/mob/living/carbon/proc/forceFed(var/obj/item/weapon/reagent_containers/food/toEat, mob/user, fullness)
+	if (fullness <= (550 * (1 + overeatduration / 1000)))
+		visible_message("<span class='warning'>[user] attempts to force [src] to [toEat.apply_method] [toEat].</span>")
+	else
+		visible_message("<span class='warning'>[user] cannot force anymore of [toEat] down [src]'s throat.</span>")
+		return 0
+	if(!do_mob(user, src))
+		return 0
+	forceFedAttackLog(toEat, user)
+	visible_message("<span class='warning'>[user] forces [src] to [toEat.apply_method] [toEat].</span>")
+	return 1
+
+/mob/living/carbon/proc/forceFedAttackLog(var/obj/item/weapon/reagent_containers/food/toEat, mob/user)
+	attack_log += text("\[[time_stamp()]\] <font color='orange'>Has been fed [toEat.name] by [user.name] ([user.ckey]) Reagents: [toEat.reagentlist(toEat)]</font>")
+	user.attack_log += text("\[[time_stamp()]\] <font color='red'>Fed [toEat.name] to [name] ([ckey]) Reagents: [toEat.reagentlist(toEat)]</font>")
+	log_attack("[user.name] ([user.ckey]) fed [name] ([ckey]) with [toEat.name] Reagents: [toEat.reagentlist(toEat)] (INTENT: [uppertext(user.a_intent)])")
+	if(!iscarbon(user))
+		LAssailant = null
+	else
+		LAssailant = user
+
+
+/*TO DO - If/when stomach organs are introduced, override this at the human level sending the item to the stomach
+so that different stomachs can handle things in different ways VB*/
+/mob/living/carbon/proc/consume(var/obj/item/weapon/reagent_containers/food/toEat)
+	if(!toEat.reagents)
+		return
+	if(toEat.consume_sound)
+		playsound(loc, toEat.consume_sound, rand(10,50), 1)
+	if(toEat.reagents.total_volume)
+		toEat.reagents.reaction(src, toEat.apply_type)
+		spawn(0)
```

Comment:
@Crazylemon64 
That's a good question. It was in the originals, and I wasn't 100% about taking it out. I'll be glad if I can take it out though, as I had to put in spawns to counteract it when the items were finished, otherwise they got deleted before the reagents were transferred